build.gradle

```diff
@@ -117,6 +117,8 @@ allprojects {
     junitVersion = '4.12'
     mockitoVersion = '1.10.19'
     hadoopVersion = '2.7.7'
+    gcsHadoop2ConnectorVersion='hadoop2-2.0.0'
+    gcsHadoop3ConnectorVersion='hadoop3-2.0.0'
```

Comment:
Why do we need `gcsHadoop3ConnectorVersion` ?

Reply:
Since there are separate versions of the connector from Google for Hadoop2 and Hadoop3, I have defined 2 different identifiers. We don't need gcsHadoop**3**ConnectorVersion now, but we will need it when we start supporting Hadoop 3.x

Reply:
Can't we do away with the Hadoop3ConnectorVersion for now and add it when we have the related support in place? We might have to remove the Hadoop2ConnectorVersion at that time.